### PR TITLE
[RISCV] Bump Zaamo and Zalrsc to version 1.0

### PR DESCRIPTION
**Pull request by 4vtomat**
The ratified information can be found here:
https://wiki.riscv.org/display/HOME/Ratified+Extensions

### CODE REVIEW

**Comment by topperc** [comment]
Need to update RISCVUsage.rst and ReleaseNotes.rst

**Comment by 4vtomat** [comment]
> Need to update RISCVUsage.rst and ReleaseNotes.rst

Is it going to be cherry-picked to release branch?

**Comment by topperc** [comment]
> > Need to update RISCVUsage.rst and ReleaseNotes.rst
> 
> Is it going to be cherry-picked to release branch?

No. We just try to update the ReleaseNotes proactively for LLVM 19.

**Comment by 4vtomat** [comment]
> > > Need to update RISCVUsage.rst and ReleaseNotes.rst
> > 
> > 
> > Is it going to be cherry-picked to release branch?
> 
> No. We just try to update the ReleaseNotes proactively for LLVM 19.

Got it!

**Review by wangpc-pp — APPROVED** [review]
LGTM. Thanks!

File: clang/test/Preprocessor/riscv-target-features.c [hunk 1]
```diff
@@ -1554,13 +1554,13 @@
 // CHECK-ZVKT-EXT: __riscv_zvkt 1000000{{$}}
 
 // Experimental extensions
-// RUN: %clang --target=riscv32 -menable-experimental-extensions \
-// RUN:   -march=rv32i_zaamo0p2 -E -dM %s \
+// RUN: %clang --target=riscv32 \
```
[the remark this answers is not in the document]
Comment (4vtomat) [reply clang/test/Preprocessor/riscv-target-features.c:1557]: Oh, I see. Thanks for catching this!

**Pull request merged**